### PR TITLE
Fixing the Cookie Settings link

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Added the COOKIE_SETTINGS_URL environment variable. This should allow the 'Cookie Settings' button that is displayed in the cookie banner to link to the Census website's cookie settings page.

### How to review 
To test locally, add the COOKIE_SETTINGS_URL with a link to your environment variables and check that the button works.

### CODE REVIEW

**Comment by berroar** [comment]
[Cookie setting button not linked to anything](https://trello.com/c/HCPiXBwn/3351-cookie-setting-button-not-linked-to-anything)

File: README.md [hunk 1]
```diff
@@ -377,6 +377,7 @@ The following env variables can be used
 | EQ_NEW_RELIC_ENABLED                      | False                 | Enable New Relic monitoring                                                                   |
 | NEW_RELIC_LICENSE_KEY                     |                       | Enable new relic monitoring by supplying a New Relic licence key                              |
 | NEW_RELIC_APP_NAME                        |                       | The name to display for the application in New Relic                                          |
+| COOKIE_SETTINGS_URL                       |                       | URL for the Census Webstie Cookie Settings page                                               |
```
Comment (MebinAbraham) [inline README.md]:
```suggestion
| COOKIE_SETTINGS_URL                       |                       | URL for the Website Cookie Settings page                                               |
```

Comment (ajmaddaford) [inline README.md]: This should be `EQ_COOKIE_SETTINGS_URL` to be consistent with the other EQ specific environment variables.

File: templates/layouts/_base.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
   {{
     onsCookiesBanner({
         "html": _('We use cookies to improve your experience of census.gov.uk'),
-        "secondaryButtonUrl": '#'
+        "secondaryButtonUrl":  cookie_settings_url
```
Comment (MebinAbraham) [inline templates/layouts/_base.html]:
```suggestion
        "secondaryButtonUrl": cookie_settings_url
```

File: k8s/deploy_app.sh [hunk 3]
```diff
@@ -19,4 +19,5 @@ helm tiller run \
     --set googleTagManagerAuth=${GOOGLE_TAG_MANAGER_AUTH} \
     --set googleTagManagerPreview=${GOOGLE_TAG_MANAGER_PREVIEW} \
     --set image.repository=${DOCKER_REGISTRY}/eq-survey-runner \
-    --set image.tag=${IMAGE_TAG}
+    --set image.tag=${IMAGE_TAG} \
+    --set cookieSettingsUrl=${COOKIE_SETTINGS_URL} \
```
Comment (rmccar) [inline k8s/deploy_app.sh]: Does the trailing slash need to be on here?

**Review by ajmaddaford — APPROVED** [review]

**Review by boxadesign — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**